styx-cli/src/main/java/com/spotify/styx/cli/PrettyCliOutput.java

```diff
@@ -94,15 +93,29 @@ public void printEvents(List<EventInfo> eventInfos) {
                                                       eventInfo.info())));
   }
 
+  private static final String BACKFILL_FORMAT = "%28s  %7s  %8s  %10s  %9s  %11s  %-13s  %-13s  %-13s  %s  %s";
+
   @Override
   public void printBackfill(Backfill backfill) {
     final Partitioning partitioning = backfill.partitioning();
     final WorkflowId workflowId = backfill.workflowId();
+    final int remaining = rangeOfInstants(backfill.nextTrigger(), backfill.end(), partitioning).size();
+    final int total = rangeOfInstants(backfill.start(), backfill.end(), partitioning).size();
+    final String state;
+    if (backfill.halted()) {
+      state = "HALTED";
+    } else if (backfill.allTriggered()) {
+      state = "DONE";
```

Comment:
`ALL_TRIGGERED`? Seems too long. `DONE` is a little bit missing leading. Wording problem again. Hmm..

Reply:
A second thought. It might be OK as `ALL TRIGGERED`, because we have `styx trigger` as a command.